src/main/kotlin/no/nav/medlemskap/routes/EvalueringRoute.kt

```diff
@@ -176,6 +209,14 @@ private suspend fun hentPersonhistorikkFraPdl(services: Services, fnr: String, c
     }
 }
 
+private suspend fun hentPDLPersonhistorikkForFamilieListe(personhistorikkRelatertPerson: List<PersonhistorikkRelatertPerson>, services: Services, callId: String) : List<Personhistorikk?> {
```

Comment:
Denne kan gjøre enklere hvis du skriver det mer funksjonelt, som dette:
`personhistorikkRelatertPerson.map { hentPersonhistorikkFraPdl(services, it.ident, callId) }`